### ISSUE TITLE
gossip: use `query_channel_range` of last 6 blocks instead of 24 hours of gossip at startup/

### ISSUE THREAD

**Issue by rustyrussell** [ElementsProject/lightning#2992]
Depends on: #2991

**Comment by rustyrussell** [comment]
AI cleanup of old issues.  If wrong, please open a new issue.


Triage decision summary:
Proposes optimization to use query_channel_range for last 6 blocks instead of 24-hour gossip at startup. Issue from 2019 with no updates; likely implemented or superseded by subsequent Core Lightning development.
